.github/workflows/ci.yml

```diff
@@ -44,6 +44,6 @@ jobs:
     - name: golangci-lint
       uses: golangci/golangci-lint-action@032fa5c5e48499f06cf9d32c02149bfac1284239
       with:
-        args: -E=goimports
+        args: -E=goimports --timeout 2m0s
```

Comment:
Thanks, we should put this change in a separate PR.

Reply:
Raised a separate PR: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1113